### PR TITLE
Update theme & logo specs

### PR DESCRIPTION
**Pull request by marionf**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Improve specs for theme & logo page, improve the specifications for the multistore

### CODE REVIEW

File: back-office/design/themes-and-logo/theme-logo.md [hunk 1]
```diff
@@ -1,32 +1,39 @@
...
+In the menu there are 3 tabs: 
 
-After click on choose layouts button, we end up on a different page: Design &gt; Theme & Logo &gt; Choose layouts
+- Theme and logo (current page) 
+- "Pages configuration" and "advanced customization" who come from the module Theme customization. IMPROVEMENT to do : If the module is disabled, hide those 2 pages from the menu \(on 1.7.5 it’s hidden\) Configuration of home page Advanced Personnalisation
 
-### Logo and favicons
+## Adaptation to Right-to-Left languages
+
+This section appears if i have language who support RTL on my store \(like Arabic or Hebrew\) even if it’s not the language selected.
```
Comment (LouiseBonnard) [inline back-office/design/themes-and-logo/theme-logo.md:32]:
```suggestion
This section appears if I have RTL languages on my store, like Arabic or Hebrew, even if they are not the default language selected.
```

File: back-office/design/themes-and-logo/theme-logo.md [hunk 2]
```diff
@@ -39,39 +46,40 @@ IMPROVEMENT to do : when i import a logo it’s not resized it must be to avoid
...
-## Theme installed
+## My themes 
 
-If i import a theme and if i select a shop a section appear on the bottom of the page after hover, a button use this theme appear,on click, the current theme is switched with the one selected and the preview of theme +used change with the theme chosen.
+All the themes imported are displayed here with a preview.
 
-If i don’t select a shop and i select the group of shop this appear instead of the theme section:
+A button "use this theme" is displayed for themes not currently used. After clicking on it, the current theme is switched with the one selected and "My current theme" is displayed for the theme used.
 
+A button "delete" is displayed for themes not currently used. After clicking on it, a modal is didsplayed: "Delete this theme ?". A button "Cancel" and a "cross" allows to cancel the deletion. A button "Yes" allows to confirm the deletion. A message "Successful deletion." is displayed to confirm the deletion.
```
Comment (LouiseBonnard) [inline back-office/design/themes-and-logo/theme-logo.md:61]:
```suggestion
A button "delete" is displayed for themes not currently used. After clicking on it, a modal is displayed: "Delete this theme ?". A button "Cancel" and a "cross" allow canceling the deletion. A button "Yes" allows confirming the deletion. A message "Successful deletion." is displayed to confirm the deletion.
```

File: back-office/design/themes-and-logo/theme-logo.md [hunk 3]
```diff
@@ -39,39 +46,40 @@ IMPROVEMENT to do : when i import a logo it’s not resized it must be to avoid
...
 
+### Visit the theme catalog 
 
-## Debug mode
+The button “Visit the theme catalog” redirects to this link: https://addons.prestashop.com/en/3-templates-prestashop?utm_source=back-office&utm_medium=theme-button&utm_campaign=back-office-EN&utm_content=download
 
-3\) Reset to defaults button: The button “reset to defaults” appear next to “choose layouts” button It allows to rest to default all modification made on the page choose layouts
+### Choose layouts 
 
+After clicking on choose layouts button, we end up on a different page: Design &gt; Theme & Logo &gt; Choose layouts
```
Comment (LouiseBonnard) [inline back-office/design/themes-and-logo/theme-logo.md:75]:
```suggestion
Clicking the "Choose layouts" button opens a different page: Design > Theme & Logo > Choose layouts. Saving redirects me to the general Design > Theme & Logo page and displays a success notification: `Successful update.`.
```

File: back-office/design/themes-and-logo/theme-logo.md [hunk 4]
```diff
@@ -39,39 +46,40 @@ IMPROVEMENT to do : when i import a logo it’s not resized it must be to avoid
...
-## RTL state
-
-If i have language who support RTL on my store \(like Arabic or Hebrew\) even if it’s not the language selected, This section appear at the bottom of the page :
-
-You can select a theme and generate a RTL stylesheet.
+### Reset to defaults
 
-If you have Multi Store + RTL language you have a yes/no button and checkbox next to “theme to adapt” and “generate rtl stylesheet” If you check no on multistore the next to element will be disabled but if you check yes on multistore or if you check the checkbox, elements will be enabled.
+The button “reset to defaults” appear next to the “choose layouts” only if the debug mode is enabled.
+It allows to rest all modifications made on the page choose layouts.
```
Comment (LouiseBonnard) [inline back-office/design/themes-and-logo/theme-logo.md:80]:
```suggestion
It resets all modifications made on the Design > Theme & Logo > Choose layouts page.
```

File: back-office/design/themes-and-logo/theme-logo.md [hunk 5]
```diff
@@ -39,39 +46,40 @@ IMPROVEMENT to do : when i import a logo it’s not resized it must be to avoid
 All logo images should be imported via the upload field, with the placeholder _Choose file(s)_ and the CTA _Browse_, all being clickable items opening the user's local files. Available logo images extensions are: .gif, .jpg, .jpeg, .jpe, .png, .svg.
 
 
-### Visit the theme catalog button
+### Multistore behavior
 
-The button “Visit the theme catalog” redirect to this link
+In a multistore context, the options listed above have [checkboxes](https://github.com/PrestaShop/PrestaShop/issues/19375) and [specific settings drop-down](https://github.com/PrestaShop/PrestaShop/issues/19327), take a look at the conditions and specs [here](https://github.com/PrestaShop/prestashop-specs/blob/master/back-office/multistoregeneralspecs.md)
```
Comment (LouiseBonnard) [inline back-office/design/themes-and-logo/theme-logo.md:51]:
```suggestion
In a multistore context, the options listed above have [checkboxes](https://github.com/PrestaShop/PrestaShop/issues/19375) and [specific settings drop-down](https://github.com/PrestaShop/PrestaShop/issues/19327), take a look at the conditions and specs [here](https://github.com/PrestaShop/prestashop-specs/blob/master/back-office/multistoregeneralspecs.md)

If I am in an all shops context or in a group of shops context, an information notification is displayed: `You must select a shop from the above list if you wish to choose a theme.`.
```

**Review by LouiseBonnard — COMMENTED** [review]
A few encoding issues and missing notifications. Otherwise, it looks okay!

File: back-office/design/themes-and-logo/theme-logo.md [hunk 6]
```diff
@@ -1,32 +1,39 @@
-# **SPECIFICATIONS -THEME & LOGO > THEME & LOGO PAGE**
+# SPECIFICATIONS -DESIGN > THEME & LOGO > THEME & LOGO
 
 
 _As a merchant, I want to be able to set up my store's look._
 
-**44 \| Design &gt; Theme & Logo &gt; Theme & Logo**
+This page refers to the issue: \#10576
```
[the remark this answers is not in the document]
Comment (marionf) [reply back-office/design/themes-and-logo/theme-logo.md]: Alright :)

**Review by LouiseBonnard — APPROVED** [review]

File: back-office/design/themes-and-logo/theme-logo.md [hunk 7]
```diff
@@ -39,39 +52,41 @@ IMPROVEMENT to do : when i import a logo it’s not resized it must be to avoid
...
-If i import a theme and if i select a shop a section appear on the bottom of the page after hover, a button use this theme appear,on click, the current theme is switched with the one selected and the preview of theme +used change with the theme chosen.
+The multistore yes / no toggle allowing to select all / unselect all checkboxes should be removed.
 
-If i don’t select a shop and i select the group of shop this appear instead of the theme section:
+## My themes 
 
+All the themes imported are displayed here with a preview.
 
-## Multistore state
+A button "use this theme" is displayed for themes not currently used. After clicking on it, the current theme is switched with the one selected and "My current theme" is displayed for the theme used.
```
Comment (Robin-Fischer-PS) [inline back-office/design/themes-and-logo/theme-logo.md:67]:
```suggestion
A button "use this theme" is displayed for themes not currently used. After clicking on it, a modal is displayed: "Use this theme?". A button "Cancel" allows canceling the switch of the theme. A button "Yes" allows switching the theme. The current theme is switched with the one selected and "My current theme" is displayed for the theme used.
```

File: back-office/design/themes-and-logo/theme-logo.md [hunk 8]
```diff
@@ -39,39 +52,41 @@ IMPROVEMENT to do : when i import a logo it’s not resized it must be to avoid
...
 
-If i activate multi store, multi store section will appear only if i click on a shop\(here: derek ortega or testshop\) or i select the group of shop\(here: default group\) \(except all shops\)
+A button "delete" is displayed for themes not currently used. After clicking on it, a modal is displayed: "Delete this theme ?". A button "Cancel" and a "cross" allow canceling the deletion. A button "Yes" allows confirming the deletion. A message "Successful deletion." is displayed to confirm the deletion.
```
Comment (Robin-Fischer-PS) [inline back-office/design/themes-and-logo/theme-logo.md:69]:
```suggestion
A button "delete" is displayed for themes not currently used. After clicking on it, a modal is displayed: "Delete this theme?". A button "Cancel" and a "cross" allow canceling the deletion. A button "Yes" allows confirming the deletion. A message "Successful deletion." is displayed to confirm the deletion.
```

Currently there is no "cross" on the confirmation modal, is this a new wanted behavior or should the spec be updated ?

**Review by Robin-Fischer-PS — CHANGES_REQUESTED** [review]

File: back-office/design/themes-and-logo/theme-logo.md (hunk 8 above)
Comment (marionf) [reply back-office/design/themes-and-logo/theme-logo.md]: You are right, I deleted it and updated the wording btw: https://github.com/PrestaShop/prestashop-specs/pull/195/commits/b85dd8ebde85f44c1baa48aedd2ad9bfdf8b2e15

**Comment by marionf** [comment]
Thank you for the review @Robin-Fischer-PS 
I applied all your suggestions

**Comment by LouiseBonnard** [comment]
Matur nuwun sanget, @marionf!

**Pull request merged**